src/Microsoft.ApplicationInsights/DataContracts/DependencyTelemetry.cs

```diff
@@ -30,6 +30,7 @@ public sealed class DependencyTelemetry : OperationTelemetry, ITelemetry, ISuppo
         public DependencyTelemetry()
         {
             this.InternalData = new RemoteDependencyData();
+            this.successFieldSet = true;
```

Comment:
i thought we wanted to set it true if underlying value InternalData.success is set to true... this also works, but makes successFieldSet more meaningless...

Reply:
@Dmitry-Matveev and I shared a concern that if the public schema was changed to `false` by default, this would cause DependencyTelemetry to return `null` by default. Because the underlying type is not nullable, it seems that a decision was made that its base value was significant.

the `this.successFieldSet` still allows `Success` to be manually set to Null.

This is certainly open for further discussion.